exchanges/deribit/deribit_websocket.go

```diff
@@ -779,6 +781,7 @@ func (d *Deribit) GetSubscriptionTemplate(_ *subscription.Subscription) (*templa
 		"channelName":     channelName,
 		"interval":        channelInterval,
 		"isSymbolChannel": isSymbolChannel,
+		"fmt":             formatPerpetualPairWithSettlement,
```

Comment:
Nit: Don't think we can call this `fmt` when it's specific to Futures.

Reply:
It calls it against every pair, are you asking me to only call it against futures?

Reply:
Nope. But I think it's misleading that it's not really format for every pair, even if it's called for all of them.
So I guess I'd vote for: Rename the template func to `fmtFuturePairs` or rename the func called to formatChannelPair.
What's catching me is the disconnect between what's in the template and the actual func name.
Can also ignore.

Reply:
[fixed](https://github.com/thrasher-corp/gocryptotrader/pull/1717/commits/9d7f1169043d7216e6d1cd18f2612cf799967662)